studio/components/to-be-cleaned/NoTableState.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react'
```

Comment:
no need to import

Reply:
oh okay, thanks for the suggestion! just made the changes 👍